src/ScrollSync.js

```diff
@@ -81,6 +76,14 @@ export default class ScrollSync extends Component {
   }
 
   render() {
-    return React.Children.only(this.props.children)
+    return (<div style={{ overflow: 'hidden', ...this.props.style }}>
```

Comment:
What if I want to use it with something different than DOM or with a different tag? Also, I don't like it is inline styles only.